packages/api-client/src/collections.ts

```diff
@@ -35,6 +35,40 @@ export const createCollection = async (params: {
   });
 };
 
+/**
+ * PUT /collections/{collectionName}/{collectionVersion}
+ *
+ * @param {Object} params              - params
+ * @param {string} params.prefix       - the prefix configured for the stack
+ * @param {Object} params.collection   - collection object to update in the database
+ * @param {Function} params.callback   - async function to invoke the api lambda
+ *                                     that takes a prefix / user payload.  Defaults
+ *                                     to cumulusApiClient.invokeApi
+ * @returns {Promise<Object>}          - the response from the callback
+ */
+export const updateCollection = async (params: {
```

Comment:
Great job adding this, just note it in the changelog

Reply:
[`8d76b1c` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/8d76b1c48461cee69ebff33bd04b3d57dc3c0796)